utilities/common.py

```diff
@@ -313,3 +318,54 @@ def list_to_string(list_str):
             "list_to_string(): cannot convert an input that is"
             " neither a single string or a list of strings"
         )
+
+_axis_defaults = {
+    "pt" : {
+        "branchName" : "{leptonsName}_pt0",
+        "default" : hist.axis.Regular(30, 26., 56., name="pt", flow=False),
+    },
+    "eta" : {
+        "branchName" : "{leptonsName}_eta0",
+        "default" : hist.axis.Regular(48, -2.4, 2.4, name="eta", flow=False),
+    },
+    "mt" : { 
+        "branchName" : "transverseMass",
+        # Should probably be variable
+        "default" : hist.axis.Regular(120, 0, 120, name="mt") 
+    },
+    "ptfake" : {
+        "branchName" : "{leptonsName}_pt0",
+        "default" : hist.axis.Regular(6, 26., 56., name="pt", flow=False),
+    },
+    "etafake" : {
+        "branchName" : "{leptonsName}_eta0",
+        "default" : hist.axis.Regular(24, -2.4, 2.4, name="eta", flow=False),
+    },
+}
+
+def get_nominal_axes_and_banches(axis_names, nbins, low_edges, high_edges, leptonsName="goodMuons"):
+    logger = logging.child_logger(__name__)
+
+    axes = []
+    branches = []
+
+    if not all(x not in axes or x+"fake" not in axes for x in ["pt", "eta"]):
+        logger.warming("Missing pt or eta axis. Will not be able to compute fake estimate correctly!")
```

Comment:
Hmm I find this not ideal but have no better idea at the moment, you can think about it, otherwise OK. The fake estimation is wmass specific, no fakes in wlike/Z. Also, could be to use fakes in pt only, for testing or so.

Reply:
Yeah, true. My concern was to make sure someone runs mT they should realise including pt and eta axes is needed.

Reply:
Maybe moving this check into mw histmaker?